docs/_templates/autosummary/class.rst

```diff
@@ -0,0 +1,31 @@
+{% extends "!autosummary/class.rst" %}
```

Comment:
Should we have a template for general functions, IO, etc.?

Reply:
I think we can leave default templates for functions since we list functions manually (we need to rewrite class template to insert `:toctree:` directive to present all class methods in the side bar, not only class).